plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -64,11 +64,13 @@ type MQTTConsumer struct {
 	ConnectionTimeout      config.Duration      `toml:"connection_timeout"`
 	MaxUndeliveredMessages int                  `toml:"max_undelivered_messages"`
 	parser                 parsers.Parser
-	// Legacy metric buffer support; deprecated in v0.10.3
-	MetricBuffer      int
+
+	MetricBuffer      int `toml:"metric_buffer" deprecated:"0.10.3;2.0.0;option is ignored"`
```

Comment:
What does the second semi-colon section (i.e. `2.0.0;`) mean and/or do?

Reply:
The second semi-colon section is optional and specifies a removal version. By default it's the next major version, but sometimes (like in this case) we need to bend this rule.